packages/odyssey-react/scripts/buildCss.js

```diff
@@ -23,8 +23,6 @@ const scssSource = [
   "abstracts/mixins",
   "abstracts/tokens",
   "base/reset",
-  "base/typography-global",
-  "base/typography-text",
```

Comment:
This is a good start, but we'll need to go much further!

This entire file `buildCss.js`, the corresponding scripts that use it `build:css` and a handful of `devDependencies` in the the corresponding [package.json](https://github.com/okta/odyssey/blob/develop/packages/odyssey-react/package.json) file should all be removed as well i.e. `postcss`, `postcss-load-config` and `sass`.

We should also remove all mention of including the deprecated global stylesheet within the [README](https://github.com/okta/odyssey/blob/develop/packages/odyssey-react/README.md#getting-started). This commit should be title as a `BREAKING CHANGE` under conventional commit format as well to describe the need for downstream consumers to remove their import of that global sheet which no longer will exist in our next release.

Reply:
Done!